eth-bridge/relay/README.md

```diff
@@ -0,0 +1,69 @@
+# Liberland Bridge Relay
+
+## Overview
+
+Liberland Bridge Relay is a part of Substrate <-> ETH bridge designed for
```

Comment:
is there a eed for it to be a same project as liberland_substrate? Seems like it would be best if its a separate repo? Lmk your thoughts and we ll create or not create another repo as needed

Reply:
Nope, no need. We can have separate repos. Especially having relay in separate repo would make releases easier (as they wouldn't be mixed with node releases on GitHub).